app/view_objects/missing_data_view.rb

```diff
@@ -35,6 +37,12 @@ def missing_data?
 
   attr_reader :form_instance, :missing_fields
 
+  def get_link_anchor(field, index)
+    return "##{get_display_name(field).parameterize}" if missing_fields.size == 1
+
+    "##{get_display_name(field).parameterize}-#{index}"
```

Comment:
I know at the moment, only the degrees could have invalid information but won't this generate for eg `#nationality-{index}` which won't anchor properly to the `nationality` row in the personal details summary?

Reply:
The guard clause above takes care of this case: https://github.com/DFE-Digital/register-trainee-teachers/pull/1251/files#diff-4f6c18f6b5e74e10a9a59f281d83e54214852c44895c4336adc290cf6c343befR41
(Note that the `missing_fields` has been converted to a nested array now, so that only in the case of multiple degrees it will add the index suffix.

For all other cases it is expected to retain the original version, i.e: `#nationality`, `#first-names` etc